modules/@shopify/checkout-sheet-kit/android/src/main/java/com/shopify/reactnative/checkoutsheetkit/ShopifyCheckoutSheetKitModule.java

```diff
@@ -77,12 +77,12 @@ public void removeListeners(Integer count) {
   @ReactMethod
   public void present(String checkoutURL) {
     Activity currentActivity = getCurrentActivity();
-    if (currentActivity instanceof ComponentActivity) {
+    if (currentActivity != null) {
```

Comment:
Wondering if this is safe.  `getCurrentActivity()` returns an `Activity`.

`ComponentActivity` is a subclass of Activity, but could `getCurrentActivity()` return another subclass and the cast below (`(ComponentActivity) currentActivity`) may fail leading to a `ClassCastException`.